.testcafe.browser.js

```diff
@@ -0,0 +1,7 @@
+const rootConfig = require('./.testcaferc.json');
+
+module.exports = {
+    ...rootConfig,
+    browsers: ['edge', 'firefox', 'chrome'],
+    concurrency: 1
```

Comment:
What are your thoughts on increasing the concurrency value to 3 (edge, firefox, chrome) instead of 1? Would such parallelization work for CI?

Reply:
The default is 3, and it was failing in Windows